header.php

```diff
@@ -85,32 +85,40 @@
     <div class="container">
       <div class="row font-size-mid">
 
-        <div class="col col-s-4">
-          <h1 class="u-visuallyhidden"><?php bloginfo('name'); ?></h1>
+        <div class="col col-s-4 margin-bottom-tiny">
```

Comment:
by default `.col` has margin-bottom-small. so im just overriding for these elems